.github/workflows/publish-c-apidocs.yml

```diff
@@ -15,16 +15,35 @@ jobs:
           sudo apt-get install libclang-cpp9
           wget https://www.doxygen.nl/files/doxygen-1.9.2.linux.bin.tar.gz
           tar xvzf doxygen-1.9.2.linux.bin.tar.gz
+      - name: Set vars
+        id: vars
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+      - name: Check outputs
+        run: echo ${{ steps.vars.outputs.sha_short }}
       - name: Run doxygen
```

Comment:
Are these setting vars and checking outputs, or just printing info to stdout?

Reply:
It's a check that the right SHA got set. I can remove this step

Reply:
It doesn't need to be removed - just the naming is confusing given it's just printing values and not directly setting/checking anything. Adding a comment to explain or renaming would be fine as well.